src/actions/wallet.js

```diff
@@ -11,10 +14,9 @@ export function selectWallet(wallet) {
   };
 }
 
-export function sendMoney(wallet, amount, toEthAddress, message) {
+export function sendMoney(amount, toEthAddress, message) {
```

Comment:
message should be removed since we don't pass additional arg's to an transaction when sending ETH (it's expensive as hell)